micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -43,6 +45,9 @@
 @NonNullApi
 @NonNullFields
 public class ExecutorServiceMetrics implements MeterBinder {
+
+    private final static InternalLogger log = InternalLoggerFactory.getInstance(JvmGcMetrics.class);
```

Comment:
Should not this be `ExecutorServiceMetrics` instead of `JvmGcMetrics`?
```suggestion
    private final static InternalLogger log = InternalLoggerFactory.getInstance(ExecutorServiceMetrics.class);
```

Reply:
@jonatan-ivanov Thanks! Fixed in ade0ef9a93e0f246475e385399724c6b53f0c22b.